workflows/gasclustering.nf

```diff
@@ -67,13 +67,32 @@ def prepareFilePath(String filep){
 }
 
 workflow GASCLUSTERING {
-
+    ID_COLUMN = "sample_name"
+    ID_COLUMN2 = "sample"
```

Comment:
I would recommend changing `ID_COLUMN` and `ID_COLUMN2` to something more like `SAMPLE_NAME_COLUMN` and `SAMPLE_COLUMN`, or `ID_SAMPLE_NAME `and `ID_SAMPLE`, so that's it's more clear the difference between the two when reading the code.

Reply:
Also, is `ID_COLUMN` used within the code? If it's not, it should probably be removed and maybe that changes the name to `ID_HEADER` or something?

Reply:
Good catch `ID_COLUMN` is a vestige from some testing. Well, really I should remove `ID_COLUMN2`, and replace with `ID_COLUMN = "sample"`.
It goes into the formatting of the `metadata_headers` channel:
```
metadata_headers = Channel.of(
        tuple(
            ID_COLUMN2,
            params.metadata_1_header, params.metadata_2_header,
            params.metadata_3_header, params.metadata_4_header,
            params.metadata_5_header, params.metadata_6_header,
            params.metadata_7_header, params.metadata_8_header)
        )
```

I think I will change it to `SAMPLE_HEADER` so it fits nicely in the channel.